File: CoreJava/man-boy-app/src/com/techlabs/inheritance/Boy.java
package com.techlabs.inheritance;

public class Boy extends Man {
	public void walk() {
		System.out.println("Boy is Walking");
	}

	@Override
	public void play() {
		System.out.println("Boy is playing");
	}

	public Boy() {
		System.out.println("Boy created");
	}
}

File: C#/OOP/InheritanceExample/InheritanceExample/Boy.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace InheritanceExample
{
    class Boy : Man
    {
        public void walk() {
		Console.WriteLine("Boy is Walking");
	}

	
	public override void play() {
		Console.WriteLine("Boy is playing");
	}

	public Boy() {
		Console.WriteLine("Boy created");
	}
    }
}
